pages/docs/features/inngest-functions/steps-workflows/sleeps.mdx

```diff
@@ -163,17 +161,15 @@ Check out the [`step.sleep_until()` Python reference.](/docs/reference/python/st
 
 </GuideSection>
 
-
-
 </GuideSelector>
 
-
-
-
 <Callout variant={'info'}>
-**About: Maximum sleep duration**
+  **Sleeps and trace/log history**
 
-Functions can sleep for a year (seven days for the [free tier plans](/pricing)).
-</Callout>
+  You may notice that Inngest Cloud's Function Runs view doesn't show function runs that use sleeps longer than your [Inngest plan's](/pricing) trace & log history limit, even though the functions are still sleeping and will continue to run as expected. **This is a known limitation** in our current infrastructure and we're working to improve it.
```

Comment:
**suggestions**:
* Add ref tag
* Clarify that it's not an infra limitation, it's a product limitation. We don't want to give the idea that our infrastructure cannot handle this, it's a UI/UX issue.

```suggestion
  You may notice that Inngest Cloud's Function Runs view doesn't show function runs that use sleeps longer than your [Inngest plan's](/pricing?ref=docs-sleeps) trace & log history limit, even though the functions are still sleeping and will continue to run as expected. **This is a known limitation** in our current dashboard and we're working to improve it.
```